R/ec2.R

```diff
@@ -0,0 +1,22 @@
+#' @import httr
+#' @importFrom jsonlist fromJson
+AWS_METADATA = "http://169.254.169.254/latest/meta-data/iam/security-credentials/" 
+
+#' @export
+ec2_roles <- function() {
+    r <- GET(AWS_METADATA)
+    out <- strsplit(content(r, "text"))
```

Comment:
It's worth noting this API doesn't return JSON, it will return a newline-delimited list.